Systems/fuel.xml

```diff
@@ -368,7 +368,7 @@ Implementation notice:
             <input>fuel/pump-flow-rate-pps</input>
             <input>-/systems/fuel/tank[5]/leak-flow-rate-pps</input>
             <input>-fuel/manifold-dissipate-rate-pps</input>
-            <input>-/fdm/jsbsim/propulsion/engine/fuel-flow-rate-pps</input>
+            <!--    <input>-/fdm/jsbsim/propulsion/engine/fuel-flow-rate-pps</input>   NOT NECESSARY: engine sucks already (see feed command) -->
```

Comment:
Tested by using up a complete tank at various power settings at 32x simulation rate.
when the engine stopped, it showed ~530 lbs used, equalling my initial filling in the system.
=> There is no leak anymore :)